package.json

```diff
@@ -5,7 +5,8 @@
   "main": "index.js",
```

Comment:
I've seen projects that define `"types": "index.d.js" property in `package.json`. It's also used in the [Typescript docs for publishing](https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html). The compiler probably defaults to the `.d.ts` file with the same name, or to `index.d.ts` though.

Reply:
I have added the `types` property. In this case it is not necessary, but seems like TypeScript documentation advises to add it: https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html#including-declarations-in-your-npm-package